cli/mainrt.rs

```diff
@@ -79,6 +80,8 @@ fn main() {
     match standalone {
       Ok(Some(future)) => {
         let (metadata, eszip) = future.await?;
+        util::logger::init(metadata.log_level);
+        load_env_variables_from_env_file(metadata.env_file.as_ref());
```

Comment:
Sorry, just looking at this PR in detail. I'm not so sure about just providing a reference to the file because the file won't necessarily exist at runtime. Probably we need to embed the env variables into the executable and warn that we're doing it.

Reply:
Hello @dsherret ,

I made changes to embed the env variables from the env file into the generated executable and to warn the user about this action.
When running the executable, it will load the embedded variables as env variables.